README.md

```diff
@@ -76,9 +76,11 @@ Currently [`XcodeServerSDK`][xcodeserversdk] provides frameworks for the followi
 
 | Platform | Minimum Version | Carthage | CocoaPods |
 |:--:|:--:|:--:|:--:|
-|OS X | 10.10| √ | √ |
-|iOS | 8.0| √ | √ |
-|ᴡᴀᴛᴄʜ | 2.0 | √ | √ |
+|OS X | 10.10| :white_check_mark: | :white_check_mark: |
+|iOS | 8.0| :white_check_mark: | :white_check_mark: |
+|ᴡᴀᴛᴄʜ | 2.0 | :white_check_mark: | :no_entry: * |
+
+*: _CocoaPods still doesn't support watchOS, but its working on it._
```

Comment:
Let's add this as a sign of a good will: https://github.com/CocoaPods/CocoaPods/pull/3681